apps/src/templates/UnsafeRenderedMarkdown.jsx

```diff
@@ -3,29 +3,61 @@ import React from 'react';
 
 import Parser from '@code-dot-org/redactable-markdown';
 
+import remarkRehype from 'remark-rehype';
+import rehypeRaw from 'rehype-raw';
+import rehypeSanitize from 'rehype-sanitize';
+import rehypeReact from 'rehype-react';
+import defaultSanitizationSchema from 'hast-util-sanitize/lib/github.json';
+
 import expandableImages from './plugins/expandableImages';
 import xmlAsTopLevelBlock from './plugins/xmlAsTopLevelBlock';
-import stripStyles from './plugins/stripStyles';
 import externalLinks from './plugins/externalLinks';
 
-const commonPlugins = [xmlAsTopLevelBlock, expandableImages];
+// create custom sanitization schema as per
+// https://github.com/syntax-tree/hast-util-sanitize#schema
+// to support our custom syntaxes
+const schema = Object.assign({}, defaultSanitizationSchema);
+
+// Add support for expandableImages
+schema.tagNames.push('span');
+schema.attributes.span = ['dataUrl', 'className'];
 
-const remarkParser = Parser.create();
-remarkParser.parser.use(commonPlugins);
-remarkParser.compilerPlugins.push(stripStyles);
+// Add support for Blockly XML
+schema.tagNames.push(
+  'block',
+  'functional_input',
+  'mutation',
+  'next',
+  'statement',
+  'title',
+  'value',
+  'xml'
+);
```

Comment:
Will be cool to be able to hand this off to a dedicated Blockly component in the future, instead of rendering it as a post-React-render pass!